charts/poktroll-sequencer/templates/deployment.yaml

```diff
@@ -81,9 +81,7 @@ spec:
           command: ["/bin/sh", "-c"]
           args: 
             - |
```

Comment:
Don't we have to specify the block time via a flag as well?

Reply:
@Olshansk we can, but we don't have to. This chart is for local development, so default block times work for us.